lib/utils/getServerlessConfigFile.js

```diff
@@ -6,7 +6,37 @@ const path = require('path');
 const fileExists = require('./fs/fileExists');
 const readFile = require('./fs/readFile');
 
+const parseJsConf = fullPath =>
+  BbPromise.try(() => {
+    // use require to load serverless.js
+    // eslint-disable-next-line global-require
+    const configExport = require(fullPath);
+    // In case of a promise result, first resolve it.
+    return configExport;
+  }).then(config => {
+    if (_.isPlainObject(config)) {
+      return config;
+    }
+    throw new Error('serverless.js must export plain object');
+  });
+
+const explicitFile = fullPath =>
+  fileExists(fullPath).then(() => {
+    switch (path.extname(fullPath).replace('.', '')) {
+      case 'json':
+      case 'yml':
+      case 'yaml':
+        return readFile(fullPath);
+      case 'js':
+        return parseJsConf(fullPath);
+      default:
+        return '';
+    }
+  });
+
 const getServerlessConfigFile = _.memoize((srvcPath) => {
+  if (srvcPath && !!path.extname(srvcPath)) return explicitFile(srvcPath);
```

Comment:
I would prefer adding a 2nd argument to `getServerlessConfigFile` rather than conflating service path and config path in one variable